### PR TITLE
Adding compliance playbook

### PR DESCRIPTION
**Pull request by ymao1**

### CODE REVIEW

File: compliance/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+- name: Ensure additional security packages installed
+  yum:
+    name:
+      - aide
+      - psacct
+      - postfix
+    state: installed
+  tags: [ compliance, yum ]
+
+- name: Ensure additional security packages installed for Amazon Linux
```
Comment (tequalsme) [inline compliance/tasks/main.yml]: Based on the task name, should this have a `when: ansible_distribution == "Amazon"`?
It looks like these are selinux packages and not specific to Amazon Linux, so maybe just update the name.

**Comment by tequalsme** [comment]
Is there a more descriptive name than compliance that we can call this?

**Comment by tequalsme** [comment]
Can you please add a README like the other roles have, describing what the role does and what if any required/optional variables it uses? Otherwise it looks good!

**Comment by ymao1** [comment]
> Can you please add a README like the other roles have, describing what the role does and what if any required/optional variables it uses? Otherwise it looks good!

I have added a readme

**Comment by ymao1** [comment]
> Is there a more descriptive name than compliance that we can call this?

nist_compliance? would that be better?

File: compliance/tasks/main.yml (hunk 1 above)
Comment (ymao1) [reply compliance/tasks/main.yml]: Verified with Tom that they should be valid for centos as well so I updated the name.

**Comment by tequalsme** [comment]
nist_compliance sounds good to me

**Comment by ymao1** [comment]
> nist_compliance sounds good to me

done!

**Pull request merged**